editoast/src/models/rolling_stock.rs

```diff
@@ -89,9 +85,8 @@ pub struct RollingStock {
     #[model(uom_unit = "units::second::option")]
     pub raise_pantograph_time: Option<Time>,
     pub version: i64,
-    #[schema(value_type = Vec<String>)]
-    #[model(remote = "Vec<Option<String>>")]
-    pub supported_signaling_systems: RollingStockSupportedSignalingSystems,
+    #[model(json)]
+    pub supported_signaling_systems: Vec<RollingStockSupportedSignalingSystem>,
```

Comment:
Same as above, maybe a `HashSet`?


```suggestion
    pub supported_signaling_systems: HashSet<RollingStockSupportedSignalingSystem>,
```